commands/run/README.md

```diff
@@ -57,7 +58,9 @@ $ lerna run watch --stream
 
 ### `--parallel`
 
-Similar to `--stream`, but completely disregards concurrency and topological sorting, running a given command or script immediately in all matching packages with prefixed streaming output. This is the preferred flag for long-running processes such as `npm run watch` run over many packages.
+Similar to `--stream`, but completely disregards concurrency and topological sorting, running a given command or script
```

Comment:
probably not affecting the rendering of the readme, but this newline didn't seem intentional